WeakAuras/GenericTrigger.lua

```diff
@@ -1781,6 +1789,21 @@ function GenericTrigger.Add(data, region)
   else
     Private.AuraWarnings.UpdateWarning(data.uid, "spammy_event_warning")
   end
+
+  if warnAboutDeprecatedInstanceFilters then
+    Private.AuraWarnings.UpdateWarning(data.uid, "deprecated_instance_filter", "warning",
```

Comment:
I don't think we should immediately put an aura warning out for this. From experience those create quite bit of friction